support/upvote-moonlightbot.md

```diff
@@ -1,18 +1,59 @@
+3. Some people are full of hatred, and they like to attack small bot developers like us. But your votes make us more resilient and allow us to continue offering the service.
 
-[![Bots for Discord](https://discords.com/bots/api/bot/314110696071888896/widget)](https://discords.com/bots/bot/314110696071888896/vote)
-[![discordbots.co](https://api.discordbots.co/bot/314110696071888896/widget)](https://api.discordbots.co/bot/314110696071888896/)
+## Perks/Rewards for voting
 
-[![MoonlightBot](https://bots.ondiscord.xyz/bots/314110696071888896/embed?showGuilds=true)](https://bots.ondiscord.xyz/bots/314110696071888896)
+For your first ever vote or after a 30 consecutive days of voting, you will receive one free week of MoonlightBot Premium. The bot will DM you with a button to claim it.
+By having voted in the past 72 hours, your max durations will be doubled. Voting also extends your premium duration by 48 hours. Be sure to vote for MoonlightBot often!
 
 {% hint style="warning" %}
-This list is not exahustive, some sites do not provide widget. For a more complete list, use the `/vote` command
+Ensure you allow Direct Messages from a mutual server so MoonlightBot can message you, otherwise you won't be able to claim your free week of Premium!
 {% endhint %}
+
+## Voting on each site
+
+### Bots for Discord
+
+To vote follow these steps:
+1. [Click here to visit the site](https://discords.com/bots/bot/314110696071888896/vote)
+2. Click the button that says "Upvote"
+3. Authorize log in with your Discord account if requested, then click Upvote again
+You have now successfully voted for MoonlightBot!
+
+### discordbots.co
+
+1. [Click here to visit the page](https://discordbots.co/bot/314110696071888896)
+2. Click "Login with Discord" in the top right corner
+3. Click the blue button that says "Authorize" to sign in
+4. Click the blue button that says "Vote"
+You have now successfully voted for MoonlightBot!
+
+### DiscordList
+
+1. [Click here to visit the page](https://discordlist.gg/bot/314110696071888896/vote)
+2. Click "Sign in to vote"
+3. Click the blue button that says "Authorize" to sign in
+4. Once signed in, it will automatically vote for you
+You have now successfully voted for MoonlightBot!
+
+### Discord Extreme List
+
+1. [Click here to visit the page](https://discordextremelist.xyz/en-US/bots/314110696071888896/upvote)
+2. Click the purple button in the middle that says "Continue" to sign in
+3. Click the blue button that says "Authorize" to sign in
+4. Click the up pointing arrow to upvote
+You have now successfully voted for MoonlightBot!
+Note: You can only vote once on this site!
+
+## Voting Reminders
+
+You can vote once on each site every 12 Hours
+When it is time to vote again, MoonlightBot will send you a reminder on Discord with the links when you can vote again, and you can repeat the procedure much more rapidly
```

Comment:
Missing punctuation.

> and you can repeat the procedure much more rapidly

This part reads awkwardly. Consider rephrasing or omitting.

Reply:
My bad for the awkward part. I meant to say that one you voted for the first time the sites should be remembering your login and it takes less steps and time to vote again.